src/gnn_reco/components/loss_functions.py

```diff
@@ -0,0 +1,30 @@
+import torch
+import numpy as np
+from torch.nn import CrossEntropyLoss
+
+
+def LogCosh(prediction, graph, target):
+    return torch.sum(torch.log(torch.cosh(((prediction[:,0]-graph[target])))))
+
+def CustomCrossEntropyLoss(prediction, graph, target):
```

Comment:
This function, and the one above, are currently not being used — but I think that's fine, if we foresee using them. This function, however, seems to be a minimal wrapper around `torch.nn.CrossEntropyLoss`, where the `graph[target].long()` bit easily could (should?) be moved to the relevant part of the model code.